MEMBER_TRAVEL_FUND.md

```diff
@@ -1,139 +1,101 @@
 
 ## Process
 
-Requests must be made prior to the start of travel.
-Requests submitted after travel has started will not be approved.
-Candidates can only ask for reimbursement of prior approved items. In case of
-new items added, the candidates must resubmit the updated request for a new approval.
+Requests must be made prior to the start of travel. Requests submitted once travel is complete will not be approved. 
 
-Review of requests will take at least 72 hours and may take considerably longer.
-Approval is not guaranteed. If a request is rejected, reimbursement will not be
-issued. It is strongly recommended that you submit your request far enough in
-advance to get approval before making relevant expenditures.
+It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures. Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures. 
 
-If you need a visa for the country in which the Node.js event you are attending
-is being held, see the document on [preparing for a visa application](./travel-visas.md).
+## Visa Tips
 
-### Request
+For all events, 3rd party or hosted Foundation events, it is recommended to apply for a visa at least 6 months prior to an event. 
 
-* Open a pull request which adds an entry to the appropriate year's file
-* Include as many fields as possible:
+### For Industry Events 
+
+If you need a visa for the country in which the Node.js event you are attending is being held, [see the document on preparing for a visa application](https://github.com/nodejs/admin/blob/master/travel-visas.md). 
+
+### What The Foundation Can Do To Help: 
+
+The Foundation [provides guidance on preparing your visa application](https://github.com/nodejs/admin/blob/master/travel-visas.md), including information on visa processing times and a checklist of items that a visa application could require.
+
+For Foundation-hosted events, Node.js Foundation provides a visa letter by The Linux Foundation (does not guarantee visa approval). Click [here](https://events.linuxfoundation.org/events/node-js-interactive-2018/attend/visa-request/) to request a visa letter.
+
+### What the Foundation Cannot Do: 
+
+Please note that unless the Foundation is specifically hosting the event, we cannot help with visa letters. The Foundation is also not able to contact consulates. When the Foundation has done this in the past, our efforts have been unsuccessful and/or we’ve lacked the expertise to be effective.  
+
+For additional information on preparing your visa application or requesting a visa letter, please email [visainfo@nodejs.org](mailto:visainfo@nodejs.org).
+
+### Requesting Travel Funds 
+
+* Open a pull request that adds an entry to the appropriate year's file
```

Comment:
### Current Formating:

* Open a pull request that adds an entry to the appropriate year's file
  * Include as many fields as possible:
    * Include the name of the event you plan to attend.
    * Include the location, dates of the event and where you will be traveling from.
    * Include the presentation you intend to give, if applicable.
    * Include the size of the stipend you wish to receive.
      * Reimbursement stipends are expected to vary with travel distance.
      * In the PR description, mention @nodejs/tsc and @nodejs/community-committee.
    * Once the final amount spent is known, update the table again with that information.
    * Certain details of your request (such as the column for date sent to finance) may be unknown initially; these can be filled in and updated during the process of reconciliation.

### Suggested Formating:

* Open a pull request that adds an entry to the appropriate year's file
  * Include as many fields as possible:
    * Name of the event you plan to attend.
    * Location and dates of the event and where you will be traveling from.
    * Presentation you intend to give, if applicable.
    * Size of the stipend you wish to receive.
  * Reimbursement stipends are expected to vary with travel distance.
  * In the PR description, mention @nodejs/tsc and @nodejs/community-committee.
  * Once the final amount spent is known, update the table again with that information.
  * Certain details of your request (such as the column for date sent to finance) may be unknown initially; these can be filled in and updated during the process of reconciliation.